### PR TITLE
Feat/group voice call

### PR DESCRIPTION
**Pull request by Arachalev**
# Fixes Issue/Linear Ticket
https://linear.app/team-plug-space/issue/FE-26/implement-group-voice-call

( This could be an existing issue or a linear ticket )

**My PR Closes #issue_number_here** or
**My PR Fixes ID-linear_ticket_number_here**
https://linear.app/team-plug-space/issue/FE-26/implement-group-voice-call

# Changes proposed

## What did you do?
I implemented the voice call feature with agora uiKit.

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos

https://user-images.githubusercontent.com/77131606/204535790-7685d62d-101d-4da9-87cb-8402684f3be8.mp4

### CODE REVIEW

**Comment by billmal071** [comment]
We need to use better icons.

File: packages/main/src/components/media-chat/VoiceCall/Join.jsx [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+// import Button from "@geist-ui/react/esm/button";
+// import Modal from "@geist-ui/react/esm/modal";
+// import Dot from "@geist-ui/react/esm/dot";
+// import Select from "@geist-ui/react/esm/select";
+// import Grid from "@geist-ui/react/esm/grid";
+// import Text from "@geist-ui/react/esm/text";
+// import Slider from "@geist-ui/react/esm/slider";
+// import Popover from "@geist-ui/react/esm/popover";
+// import { AuthContext } from "../AuthContext";
+// import AgoraRTC from "agora-rtc-sdk";
```
Comment (billmal071) [inline packages/main/src/components/media-chat/VoiceCall/Join.jsx:16]: remove if it's not needed.

**Review by billmal071 — CHANGES_REQUESTED** [review]

**Comment by Arachalev** [comment]
I've removed the commented files.

**Comment by Arachalev** [comment]
@billmal071 I've removed the commented code from the files

**Pull request merged**